src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/PendingState.java

```diff
@@ -304,23 +304,23 @@ public Predicate<Sample> samplePredicate(MetricCategory requestCategory) {
     return sample -> qualifySample(sample, requestCategory);
   }
 
-  protected boolean qualifyCase(Case kase) {
+  public boolean qualifyCase(Case kase) {
     return kase.getTests().stream().anyMatch(test -> qualifyTest(test));
   }
 
-  protected boolean qualifyTest(Test test) {
+  public boolean qualifyTest(Test test) {
     throw new IllegalStateException("Method undefined");
   }
 
-  protected boolean qualifySample(Sample sample, MetricCategory requestCategory) {
+  public boolean qualifySample(Sample sample, MetricCategory requestCategory) {
     return true;
   }
 
-  protected boolean qualifyRequisition(Requisition requisition) {
+  public boolean qualifyRequisition(Requisition requisition) {
     return true;
   }
 
-  protected Function<Case, Stream<Sample>> getFilteredSamples(MetricCategory requestCategory) {
+  public Function<Case, Stream<Sample>> getFilteredSamples(MetricCategory requestCategory) {
```

Comment:
Does this need to be `public`?

Reply:
i think some of the preliminary changes that I was trying out before rebase also got pushed. I will make it `protected`. Sorry for the confusion